airflow/operators/dagrun_operator.py

```diff
@@ -55,6 +57,14 @@ class TriggerDagRunOperator(BaseOperator):
         When reset_dag_run=False and dag run exists, DagRunAlreadyExists will be raised.
         When reset_dag_run=True and dag run exists, existing dag run will be cleared to rerun.
     :type reset_dag_run: bool
+    :param wait_for_completion: Whether or not wait for dag run completion.
+    :type wait_for_completion: bool
+    :param poke_interval: Poke internal to check dag run status when wait_for_completion=True.
```

Comment:
typo "Poke internal"? Should be "Poke interval"

Reply:
In addition, would be good to tell users what's the unit here (seconds).

Reply:
And add the default value (60) in the docstring as well please.

Reply:
will add it and fix typo :(